src/Controllers/CourseController.php

```diff
@@ -18,6 +18,34 @@
 
 class CourseController extends Controller
 {
+    public function view()
+    {
+        try {
+            $user = auth()->user();
+
+            $courseUsers = CourseUser::where('user_id', '=', $user->id)
```

Comment:
please help me figure out a more intuitive variable name for this
arigatou~

Reply:
I think since all the fields come from course, `courses` should fit this one